dataflow/flex-templates/streaming_beam/streaming_beam.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""An Apache Beam streaming pipeline example.
+
+It reads JSON encoded messages from Pub/Sub, transforms the message data and
+writes the results to BigQuery.
+"""
+
+from __future__ import absolute_import
```

Comment:
Nit: I don't think we're using `absolute_import` for anything, or is it?

Reply:
Correct, done.